src/components/shared/TreeView.js

```diff
@@ -496,12 +506,16 @@ class TreeView<
   }
 
   _onKeyDown(event: KeyboardEvent) {
-    const hasModifier = event.ctrlKey || event.altKey || event.metaKey;
-    const isArrowKey = event.key.startsWith('Arrow');
+    const hasModifier = event.ctrlKey || event.altKey;
```

Comment:
Because I removed `event.metaKey` from this boolean calculation, I'd like some MacOS user check that I'm not hijacking and preventing some well known browser or OS shortcut using Command from working. ping @gregtatum @mstange